frontend/src/api/Environment/getPivotRoleAsPartOfEnvironment.js

```diff
@@ -0,0 +1,11 @@
+import { gql } from 'apollo-boost';
+
+const getPivotRoleAsPartOfEnvironment = () => ({
```

Comment:
do we need to expose a dedicated query for it? maybe we should think about something more generic like query for returning the config?
Another option we can use 'REACT_APP_' variables for React as we do for other stuff:
REACT_APP_GRAPHQL_API={graphql_api_url}
REACT_APP_SEARCH_API={search_api_url}
REACT_APP_COGNITO_USER_POOL_ID={user_pool_id}
REACT_APP_COGNITO_APP_CLIENT_ID={app_client}
REACT_APP_COGNITO_DOMAIN={domain}
REACT_APP_COGNITO_REDIRECT_SIGNIN=https://{signin_singout_link}
REACT_APP_COGNITO_REDIRECT_SIGNOUT=https://{signin_singout_link}
REACT_APP_COGNITO_HOSTED_UI_ENABLED={'true' if cognito_hosted_ui_enabled == 'True' else 'false'}
REACT_APP_COGNITO_AUTHENTICATION_PROVIDER_NAME={cognito_authentication_provider_name}
REACT_APP_USERGUIDE_LINK=https://{user_guide_link}

Reply:
I am going to discuss it with Amr, to see if this is something we can do as part of the frontend refactoring. I foresee us having many of this kind of parameters in the future